tools/gce_setup/grpc_docker.sh

```diff
@@ -1274,8 +1274,7 @@ grpc_cloud_prod_gen_go_cmd() {
 #   cmd=$($grpc_gen_test_cmd $flags)
 grpc_interop_gen_java_cmd() {
     local cmd_prefix="sudo docker run grpc/java";
-    local test_script="/var/local/git/grpc-java/run-test-client.sh";
-    local test_script+=" --use_test_ca=true --use_tls=true"
+    local test_script="/var/local/git/grpc-java/run-test-client.sh --use_test_ca=true --use_tls=true";
```

Comment:
I see, the bunch of "node" and "csharp" tests just below the java tests misled me.

Let me know if you want they reverted, otherwise I won't bother to change it back.